From ad7a7cdbecb61e3e56f7b517311cc418c3b9d242 Mon Sep 17 00:00:00 2001
From: Volker Krause <vkrause@kde.org>
Date: Wed, 17 Jan 2024 17:56:28 +0100
Subject: [PATCH] Search for Threads::Threads

This is a workaround for linking to a static ZXing in Flatpaks (which is
needed to work around the clash between "our" ZXing and the private on
in the KDE Platform Flatpak).
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 782d9543..1423e5ac 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -66,6 +66,7 @@ set_package_properties("PhoneNumber" PROPERTIES PURPOSE "Parsing and geo-coding
 find_package(OpenSSL 1.1 REQUIRED COMPONENTS Crypto)
 find_package(OsmTools)
 set_package_properties(OsmTools PROPERTIES TYPE OPTIONAL PURPOSE "Needed only for regenereating the airport database (ie. you most likely don't need this)")
+find_package(Threads) # workaround for static ZXing in Flatpak (not needed with Qt6 anymore)
 
 if (NOT ANDROID)
     set_package_properties(KF${KF_MAJOR_VERSION}CalendarCore PROPERTIES TYPE REQUIRED)
-- 
2.42.0

